pages/common/curl.md

```diff
@@ -6,7 +6,7 @@
 
```

Comment:
I would suggest using this link https://curl.se/docs/manpage.html.

Reply:
Fixed manually.